db-engine/pom.xml

```diff
@@ -33,6 +33,16 @@
     <artifactId>db-engine</artifactId>
     <dependencies>
         <dependency>
+            <groupId>org.apache.hadoop</groupId>
```

Comment:
is there a non-vendoer specific version of these we can depend on at this level?